reframe/core/schedulers/lsf.py

```diff
@@ -94,7 +94,8 @@ def poll(self, *jobs):
             return
 
         completed = _run_strict(
-            f'bjobs -noheader {" ".join(job.jobid for job in jobs)}'
+            'bjobs -o "jobid: user:10 stat: queue:" -noheader '
+            f'{" ".join(job.jobid for job in jobs)}'
```

Comment:
I updated the PR allowing users to specify extra submission options in the config. Let me know if this works for you.

As a side note, maybe we could generalise this concept and enable it for other schedulers too, as this could be beneficial in cases such as https://github.com/EESSI/test-suite/issues/68.

Reply:
This looks good to me, although I think we should have a default that works on all systems and not require specific configuration. Also as it is today it is difficult to track exactly what the issue is when Reframe just fails to find the job because the site decided to change the default bjobs output format.